pkg/util/iptables/iptables_test.go

```diff
@@ -1046,6 +1046,8 @@ func TestRestoreAllWaitOldIptablesRestore(t *testing.T) {
 
 	// Failure.
 	err = runner.Restore(TableNAT, []byte{}, FlushTables, RestoreCounters)
+	// restore creates a file to mimic iptables lock if the runner restoreWaitFlag == 0
+	defer os.Remove(lockPath16x)
```

Comment:
This should go at line 1029 - before `runner` is ever used.  Probably warrants a comment along the lines of:

```
// the lockPath14x is a UNIX socket which is cleaned up automatically on close, but the
// lockPath16x is a plain file which is not cleaned up.
```